metaseq/cli/train.py

```diff
@@ -492,7 +492,7 @@ def _checkpoint_add_directory(basename):
     return m[1], f"checkpoint{m[3]}"
 
 
-def post_checkpoint_callback(cfg, num_updates, training_finished, filename):
+def post_checkpoint_callback(cfg, num_updates, training_finished, filename, files_to_symlink_to):
```

Comment:
if using `functools.partial` need to change order of `filename` and `files_to_symlink_to`

Reply:
wdym? https://github.com/facebookresearch/metaseq/blob/0c5db28049f105f60c28dda047fd195a78458239/metaseq/cli/train.py#L473-L475 both will need to be passed in

Reply:
See comment on readability in other thread (can go either way, nbd)